data/nat/basic.lean

```diff
@@ -859,4 +859,9 @@ lemma with_bot.add_eq_one_iff : ∀ {n m : with_bot ℕ}, n + m = 1 ↔ (n = 0 
 | (some n) (some (m + 1)) := by erw [with_bot.coe_eq_coe, with_bot.coe_eq_coe, with_bot.coe_eq_coe,
     with_bot.coe_eq_coe, with_bot.coe_eq_coe]; simp [nat.add_succ, nat.succ_inj', nat.succ_ne_zero]
 
+-- induction
+
+lemma induction_from_start (P : nat → Prop) {m} (h0 : P m) (h1 : ∀ n ≥ m, P n → P (n + 1)) : ∀ n ≥ m, P n :=
+by apply nat.less_than_or_equal.rec h0; exact h1
+
```

Comment:
Should this be tagged `@[elab_as_eliminator]` and `P` made implicit?

Reply:
Done, thanks

Reply:
how about calling this `le_induction` to match `le_rec_on` in #585 ?

Reply:
Done.